tests/test_minecraft.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from tempus_fugit_minecraft.main import Model,Window
+from tempus_fugit_minecraft.main import SAND, GRASS, BRICK, LIGHT_CLOUD, DARK_CLOUD
+
+
+
+
```

Comment:
too much space, remove extra newlines and only leave 2 between imports and the first function